update-changelog/testdata/_index.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Release notes"
+weight = 10000
++++
+
+# Release notes
+
+Here you can find detailed release notes that list everything that is included in every release as well as notices
+about deprecations, breaking changes as well as changes that relate to plugin development.
+
+- [Release notes for 8.3.2]({{< relref "release-notes-8-3-2" >}})
+- [Release notes for 8.3.1]({{< relref "release-notes-8-3-1" >}})
+- [Release notes for 8.3.0]({{< relref "release-notes-8-3-0" >}})
```

Comment:
Based on earlier comment in PR. With this update adding a test for generating changelog for 8.2.2 would be nice and it should be inserted between 8.3.0 and 8.2.1 :) Maybe can make use of markers as we do with the changelog.md file, i.e. `<!-- 8.3.3 START -->` and `<!-- 8.3.3 END -->` etc, so you don't have to parse the markdown. 

```suggestion
- [Release notes for 8.3.2]({{< relref "release-notes-8-3-2" >}})
- [Release notes for 8.3.1]({{< relref "release-notes-8-3-1" >}})
- [Release notes for 8.3.0]({{< relref "release-notes-8-3-0" >}})
- [Release notes for 8.2.1]({{< relref "release-notes-8-2-1" >}})
- [Release notes for 8.2.0]({{< relref "release-notes-8-2-0" >}})
```